lab_5_scrapper/scrapper.py

```diff
@@ -204,13 +450,78 @@ def prepare_environment(base_path: Union[pathlib.Path, str]) -> None:
     Args:
         base_path (Union[pathlib.Path, str]): Path where articles stores
     """
+    base_path.mkdir(parents=True, exist_ok=True)
+
+    for file in base_path.iterdir():
+        file.unlink(missing_ok=True)
+
+
+def clear_recursive_crawler(base_path: Union[pathlib.Path, str]) -> None:
```

Comment:
we are using more modern Python, so 
```suggestion
def clear_recursive_crawler(base_path: Path | str) -> None:
```

Reply:
Applied